mapper/mapper_test.go

```diff
@@ -77,6 +77,15 @@ var availableItems = []model.Topics{
 		ID:    "4567",
 	},
 }
+var cfg *config.Config
+
+func init() {
+	var err error
+	cfg, err = config.Get()
+	if err != nil {
+		panic("failed to get config")
```

Comment:
quick thought:
```suggestion
		panic("failed to get config" + err.Error())
```
needs the error message, too